modules/core/settings/farm_settings.links.menu.yml

```diff
@@ -0,0 +1,13 @@
+farm.settings:
+  title: Settings
+  description: Configure settings for farmOS and installed modules.
+  parent: farm.setup
```

Comment:
I think this creates an implicit dependency on `farm_setup`?

Reply:
Somewhat a dependency, yes, but does not cause any issues. If `farm_setup` is not enabled it puts `farm.settings` at the root of the main menu. But I'm coming around to just adding `farm_setup` as a dependency